pkg/containers/service.go

```diff
@@ -0,0 +1,35 @@
+package containers
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/aquasecurity/tracee/pkg/containers/runtime"
+)
+
+type runtimeInfoService struct {
+	enrichers map[runtime.RuntimeId]runtime.ContainerEnricher
```

Comment:
also here, will we have any scenario in which enricher is added(/deleted) on the fly (dynamically)

Reply:
Theoretically if we can detect the installation of a runtime while tracee is running this could be useful.